bin/iex

```diff
@@ -43,6 +43,14 @@ readlink_f () {
   fi
 }
 
+ERL="-noshell -user Elixir.IEx.CLI"
 SELF=$(readlink_f "$0")
 SCRIPT_PATH=$(dirname "$SELF")
-exec "$SCRIPT_PATH"/elixir --no-halt --erl "-noshell -user Elixir.IEx.CLI" +iex "$@"
+
+if [ -z "$HOME" ]; then
+  :
+else
+  ERL="-kernel shell_history_path \"$HOME/.elixir-history\" $ERL"
+fi
```

Comment:
Windows is giving you the %HOMEPATH% variable for use inside CMD. It doesn't not even contain a drive letter, for example it's `\Users\dimi` on my Win10 machine. %HOMEDRIVE% complements it, mine yields `C:`. So I imagine concatenating both gives the desired result.

@josevalim Do you need help translating only the highlighted part?